docs/CHANGELOG.rst

```diff
@@ -70,6 +70,7 @@ Changed
 - Use ``resolwebio/common:1.3.1`` as a base Docker image for the
   ``resolwebio/wgbs:1.2.0`` Docker image
 - Support MultiQC reports in ChIP-seq workflows
+- Use human samples for ``alignment-star`` testing in ``test_alignment.py``
```

Comment:
Break the line to be no more than 72 characters.

Reply:
I took samples from `workflow_bbduk_star_htseq` and this is the new outcome:
```
N_unmapped	0	0	0
N_multimapping	0	0	0
N_noFeature	70	77	80
N_ambiguous	0	0	0
ENSG00000279493	17	10	7
```